frontend/src/lib/components/PropertiesTable.scss

```diff
@@ -1,5 +1,28 @@
 @import '~/vars';
 
+.properties-table {
+    overflow-x: auto;
+
+    table {
+        width: 100%;
+    }
+
+    tr {
+        border-bottom: 1px solid $border;
```

Comment:
This results in an orphaned "second" border at the bottom:
<img width="370" alt="Screen Shot 2022-01-07 at 16 32 27" src="https://user-images.githubusercontent.com/4550621/148566819-0bbe2a0e-138e-42f0-b980-79435ae749b4.png">